doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='operator-deploy-pim-proc']
 = Setting Process Instance Migration configuration for the environment
 
-You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instances in a specific KIE Server. 
+You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instances in a specific {KIE_SERVER}. 
```

Comment:
For minimalism, I suggest to remove "then" from: You can then apply the migration plan to the running process instances in a specific {KIE_SERVER}.